packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -107,6 +109,8 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     ///         It is not safe to trust `ERC20.balanceOf` as it may lie.
     uint256 internal _balance;
 
+    address public balanceClaimer;
+
```

Comment:
- missing natspec
- i'd store the address casted to an interface like the rest of contracts for neatness

Reply:
+1